src/decorator/options/ValueTransformer.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Interface for objects that deal with (un)marshalling data.
+ */
+export interface ValueTransformer<P, Q> {
+
+    /**
+     * Used to marshal data when writing to
+     * the database.
+     */
+    transformTo (value: P): Q;
```

Comment:
What do you think if we name them as `to` and `from` ? Also, we should be able to do: 

```ts

 +@Entity()
 +export class Post {
 +
 +    @PrimaryGeneratedColumn()
 +    id: number;
 +
 +    @Column()
 +    title: string;
 +
 +    @Column({ type: String, transform: { from: tags => tags.join(","), to: tags => tags.split(",") } })
 +    tags: string[];
 +
 +}
```

Reply:
@pleerock Yeah, that's a lot better.